payments_service/braintree/templates/braintree/emails/subscription_charged_unsuccessfully.html

```diff
@@ -0,0 +1,38 @@
+{# subscription header is added automatically here #}
+
+    <div class="padded">
+        <div class="error">Subscription charge failed</div>
```

Comment:
Yes, as noted I moved it down to avoid header templating problems. I spoke to bram and he's cool with that.